source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -66,15 +78,29 @@ content: |
   .. image:: /figures/CSFLE_Master_Key_KMS.png
      :alt: Diagram that describes creating a master key when using a KMS provider
 
+  .. _aws-create-master-key:
+
   1. To create a master key, log into your AWS management console and create
      a new symmetric master key in the KMS section. Choose a name and
      description that helps you identify it; these fields do not affect the
      functionality or configuration.
 
   2. In the :guilabel:`Usage Permissions` step of the key generation
-     process, add the full KMS ``List`` and ``Read`` permissions to the IAM
-     user you created in the previous step. This authorizes the user to
-     encrypt and decrypt the new master key.
+     process, use the following default key policy to enable IAM policies
+     to allow access to your remote master key:
+
+     .. literalinclude:: /includes/key-policy-default.json
+        :language: json
+
+     .. note:: Key Policies and IAM policies
+
+        If you do not wish for your IAM policies to grant key access permissions,
```

Comment:
Issue: I'm not entirely sure I understand this note, or whether what it conveys is accurate. My impression from reading it is that it is stating some way to more granularly grant permissions to encrypt/decrypt using the key to individual IAM users.
Suggestion: If I understand correctly, this might communicate the message better: "If you intend to grant only specific users access to the key, you must grant access to each IAM user in the key policy. Otherwise, all IAM users can be granted access to a key through an IAM policy". If I don't understand correctly, feel free to ignore this comment.

Reply:
The note does not really pose a more granular way to grant access, but more a way to make it such that you cannot add additional users to a key (AFAICT this also makes it possible for an AWS administrator to be unable to view a key they created in a provable way).

After sitting on this for a bit, I think it may make sense to remove this and instead just link to AWS documentation. This seems like a bit of an edge case, and I'm not sure if we should discuss it. If an external reviewer thinks this explanation could benefit readers I'll add it back, but at present I think it is alright to omit it.